repose-aggregator/tests/release-verification/src/docker/deb/DELETE-ME.deb

```diff
@@ -0,0 +1,3 @@
+THIS IS NOT AN INSTALLABLE DEB PACKAGE.
```

Comment:
I did today. I'm not sure why this didn't occur to me earlier.

So I don't think `VOLUME` will work in this context. The problem is that we need to run scripts as part of building the image. Well, the `VOLUME` is also declared as part of building the image. Thus, there is no way to map the Docker `VOLUME` to a host directory before the mapped contents need to be accessed. In other words, the `docker build` command does not accept a `VOLUME` mapping.